### PR TITLE
Add rubin-libradtran feedstock.

### PR DESCRIPTION
**Pull request by erykoff**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/libradtran/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - gsl
+    - python
+  run:
+    - python
+    - gsl
+    - libnetcdf
+    - netcdf4
+    - libopenblas
+    - jpeg
+    - hdf5
```
Comment (beckermr) [inline recipes/libradtran/meta.yaml:41]: Please double check the host vs run lists here. Many of these packages have run exports and so do not need to be in run. 

Also, IIRC the blas pinning does not follow our docs.

**Review by beckermr — CHANGES_REQUESTED** [review]
Thanks for the name change. There appear to be some minor issues in the host vs run sections.

File: recipes/libradtran/meta.yaml (hunk 1 above)
Comment (erykoff) [reply recipes/libradtran/meta.yaml]: Okay, I think I've fixed the blas pinning, and I've simplified the run exports hopefully not too far.  It took some trial and error to get to the point where it did run, but I guess not in the optimal configuration.

Comment (beckermr) [reply recipes/libradtran/meta.yaml]: right ideally our linter would let you know if a package doesn't have run exports. In practice you have to go look at the feedstock to know.

**Comment by erykoff** [comment]
@mrb I think this is ready!

**Pull request merged**